cli/CMakeLists.txt

```diff
@@ -30,6 +30,12 @@ if(LIBNUMA_FOUND)
   set(QUICKSTEP_HAVE_LIBNUMA TRUE)
 endif()
 
+if (BUILD_SHARED_LIBS)
+  set(GFLAGS_LIB_NAME gflags_nothreads-shared)
+else()
+  set(GFLAGS_LIB_NAME gflags_nothreads-static)
+endif()
```

Comment:
You'd think so, but it's not! There's no common target gflags_nothreads that you can link against. It's only a problem with this particular third_party library. I think it's only fixed in the latest version of gflags (https://github.com/gflags/gflags/issues/117).